kit/ChildSession.cpp

```diff
@@ -106,37 +106,36 @@ int sendURPToClient(void* pContext /* std::function* of sendBinaryFrame */,
     std::memcpy(response, header.data(), header.size());
     std::memcpy(response + header.size(), pBuffer, nLen);
     KitSocketPoll::mainPoll->addCallback([pContext, response, responseSize]() {
-        (*(std::function<bool(const char* buffer, int length)>*)pContext)(response, responseSize);
+        static_cast<ChildSession*>(pContext)->sendBinaryFrame(response, responseSize);
     });
     return 0;
 }
 
-ChildSession::ChildSession(
-    const std::shared_ptr<ProtocolHandlerInterface> &protocol,
-    const std::string& id,
-    const std::string& jailId,
-    const std::string& jailRoot,
-    DocumentManagerInterface& docManager) :
-    Session(protocol, "ToMaster-" + id, id, false),
-    _jailId(jailId),
-    _jailRoot(jailRoot),
-    _docManager(&docManager),
-    _viewId(-1),
-    _isDocLoaded(false),
-    _copyToClipboard(false),
-    _canonicalViewId(-1),
-    _isDumpingTiles(false)
+ChildSession::ChildSession(const std::shared_ptr<ProtocolHandlerInterface>& protocol,
+                           const std::string& id, const std::string& jailId,
+                           const std::string& jailRoot, DocumentManagerInterface& docManager)
+    : Session(protocol, "ToMaster-" + id, id, false)
+    , _jailId(jailId)
+    , _jailRoot(jailRoot)
+    , _docManager(&docManager)
+    , _viewId(-1)
+    , _isDocLoaded(false)
+    , _copyToClipboard(false)
+    , _canonicalViewId(-1)
+    , _isDumpingTiles(false)
+    , m_hasURP(false)
 {
     if (std::string(std::getenv("ENABLE_WEBSOCKET_URP")) == "true")
```

Comment:
It is prolly better to get a StdlyCapsGlobal here setup on start-up of the kit; just how 'AnonymizationSalt' is setup - copy how that's done through the code - this global env. var is not really a per-instance member concept =)